From cf5bb43a9552e9e5c2d303a50e9b5e0eac45f70d Mon Sep 17 00:00:00 2001
From: Jie Chen <jie.a.chen@intel.com>
Date: Fri, 14 Feb 2014 08:50:17 +0800
Subject: [PATCH] ignore the conversion error using -fpermissive

env_chromium.cc:872:63: error: invalid conversion from
'int64_t* {aka long long int*}' to 'int64* {aka long int*}'
if (!::base::GetFileSize(CreateFilePath(fname), &signed_size))
---
 third_party/leveldatabase/leveldatabase.gyp |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/third_party/leveldatabase/leveldatabase.gyp b/third_party/leveldatabase/leveldatabase.gyp
index 49eb684..7c077e8 100644
--- a/third_party/leveldatabase/leveldatabase.gyp
+++ b/third_party/leveldatabase/leveldatabase.gyp
@@ -24,6 +24,9 @@
       'src/',
       'src/include/',
     ],
+    'cflags': [
+      '-fpermissive',
+    ],
     'conditions': [
       ['use_snappy', {
         'defines': [
-- 
1.7.9.5

